test/test_torch.py

```diff
@@ -7433,6 +7433,23 @@ def test_copy_broadcast(self, device) -> None:
         x.copy_(y)
         self.assertEqual(x[3], y)
 
+    def test_clamp(self, device):
+        x = torch.randn(100, 50)
+        y = torch.randn(100, 50)
+        z = torch.randn(100, 50)
+        ub, lb = y.max(z), y.min(z)
+
+        expect = x.max(lb).min(ub)
+        actual = x.clip(lb, ub)
+        self.assertEqual(expect, actual)
+
+        lb = lb[0]
+        ub = ub[:, :1]
+
+        expect = x.max(lb).min(ub)
+        actual = x.clip(lb, ub)
```

Comment:
The alias will be automatically tested by the OpInfo so we don't need to worry about this here